dali/kernels/imgproc/color_manipulation/brightness_contrast_gpu_test.cu

```diff
@@ -130,7 +130,12 @@ TYPED_TEST(BrightnessContrastGpuTest, setup_test) {
   KernelContext ctx;
   InListGPU<typename TypeParam::In, kNdims> in(this->input_device_, this->shapes_);
   auto reqs = kernel.Setup(ctx, in, this->brightness_, this->contrast_);
-  SUCCEED();
+  ASSERT_EQ(this->shapes_.size(), static_cast<size_t>(reqs.output_shapes[0].num_samples()))
```

Comment:
Weird indetns here.

Reply:
It's clang format. I suppose it's because EXPECT is inside `for`, while ASSERT not